js/ui/menu.d.ts

```diff
@@ -277,6 +277,16 @@ export interface dxMenuItem extends dxMenuBaseItem {
      * @type Array<dxMenuItem>
      */
     items?: Array<Item>;
+    /**
+     * @docid
+     * @public
+     */
+    url?: string;
+    /**
+     * @docid
+     * @public
+     */
+    linkAttr?: { [key: string]: any };
```

Comment:
I think we should add @default doc tag and default is {}, this way is simular for declare elementAttr option default value. In this case you need to extend _getDefaultOption() method with the new linkAttr option: 
```
linkAttr: {}
```
And we can change `itemData.linkAttr || {}` to itemData.linkAttr.